openlibrary/templates/account/borrow.html

```diff
@@ -164,7 +164,7 @@ <h1>$_("Books You're Waiting For")</h1>
         });
     });
     </script>
-    <div id="borrowTable" class="borrow collapse waitinglist">
+    <div class="borrow borrowTable collapse waitinglist">
```

Comment:
could the existing borrow class be used?
Does borrow mean something else, elsewhere?

If not, I suggest using a new class borrow-table which is more BEM-like : http://getbem.com/naming/

Reply:
I tried checking by vscode search option and I was not able to find any more instance of borrowtable in any file other than what I changed 

Will change it to borrow-table to make it bem like